dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.sh

```diff
@@ -61,7 +69,11 @@ export SITE=$(cat /etc/cluster_name)
 ############################################################
 # CALL executable job script here
 ############################################################
+<<<<<<< HEAD
+${HOMEevs}/jobs/JEVS_GLWU_WAVE_GRID2OBS_PLOTS
+=======
 ${HOMEevs}/jobs/JEVS_GLWU_PLOTS
+>>>>>>> develop
 
 #########################################################################
 # Purpose: This job creates the plots for the NFCENS wave model
```

Comment:
@SamiraArdani-NOAA In this file you have several of the merge tags (the lines with >>>>>>> and <<<<<<<) that still remain in these files.  Please clean up by removing these tags and ensuring that the lines you want to be part of these scripts remain.